agent/api/task_unix.go

```diff
@@ -149,8 +151,11 @@ func (task *Task) buildLinuxMemorySpec() (specs.LinuxMemory, error) {
 					containerMemoryLimit, task.Memory)
 		}
 	}
+
+	// Kernel expects memory to be expressed in bytes
+	memoryBytes := task.Memory * BytesPerMegabyte
```

Comment:
This shouldn't be compiling? `bytesPerMegabyte` != `BytesPerMegabyte`

Reply:
Whoops. I don't know how find/replace works. :)